client/src/components/Common/Layout/Stake/Form.scss

```diff
@@ -0,0 +1,12 @@
+.cd_setting_list_items {
+	.cd_setting_items_form {
```

Comment:
Please define css in public folder for easier  maintenance   later

Reply:
Fixed in https://github.com/CasperDash/casperdash-client/pull/33/commits/3662b136840b6db078888abaf496c0da0438471a